examples/simpleapp/.gitignore

```diff
@@ -9,3 +9,5 @@ src/main/resources/sc_settings_with_mc.conf
 .settings/
 .vscode/
 /src/main/MyOwnSettings/
+../Dockerfile
```

Comment:
these 2 lines duplicates the newly added records in `.gitignore` file.
Remove them from here.